src/Backend/Modules/CommerceMollie/Domain/PaymentMethod/MolliePaymentMethodType.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace Backend\Modules\CommerceMollie\Domain\PaymentMethod;
+
+use Backend\Modules\Commerce\Domain\OrderStatus\OrderStatus;
+use Backend\Modules\Commerce\Domain\OrderStatus\OrderStatusTransformer;
+use Backend\Modules\Commerce\Domain\PaymentMethod\PaymentMethodType;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormFactory;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class MolliePaymentMethodType extends PaymentMethodType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        parent::buildForm($builder, $options);
+
+        $builder
+            ->add('name', TextType::class, [
+                'required' => true,
+                'label' => 'lbl.Name',
+            ])
+            ->add(
+                'apiKey',
+                TextType::class,
+                [
+                    'required' => true,
+                    'label' => 'lbl.ApiKey',
+                ]
+            )
+            ->add(
+                'orderInitId',
+                EntityType::class,
+                [
+                'required' => true,
+                'label' => 'lbl.OrderInitialized',
+                'class' => OrderStatus::class,
+                'query_builder' => function (EntityRepository $er) {
+                    return $er->createQueryBuilder('i')
+                              ->orderBy('i.title', 'ASC');
+                },
+                'choice_label' => 'title',
```

Comment:
```suggestion
                    'required' => true,
                    'label' => 'lbl.OrderInitialized',
                    'class' => OrderStatus::class,
                    'query_builder' => function (EntityRepository $er) {
                        return $er->createQueryBuilder('i')
                                  ->orderBy('i.title', 'ASC');
                    },
                    'choice_label' => 'title',
```

An indentation needed?